src/Catty UITests/ScriptCollectionView/BrickCellPickerViewerTest.swift

```diff
@@ -100,41 +100,16 @@ class BrickCellPickerViewerTests: XCTestCase {
             app.pickerWheels.element.adjust(toPickerWheelValue: testValues[0])
             app.buttons[kLocalizedDone].tap()
             XCTAssert(waitForElementToAppear(app.otherElements[ACCESSABILITY_LABEL_VARIABLE + testValues[0]]).exists, "Error while changing variable for Brick: " + brick)
-        }
-    }
-
-    func testChangeVariableCancel() {
-        let bricks: [String] = [
-            kLocalizedSetVariable,
-            kLocalizedChangeVariable,
-            kLocalizedShowVariable
-        ]
-
-        app.tables.staticTexts[kLocalizedBackground].tap()
-        app.tables.staticTexts[kLocalizedScripts].tap()
-
-        for brick in bricks {
-            clearScript()
-            addBrick(label: brick, section: kLocalizedCategoryVariable, in: app)
-            if variableBrickHasValues == false {
-                addValuesForVariableBrick(brick: brick, category: kLocalizedCategoryVariable)
-                variableBrickHasValues = true
-            }
-
-            app.collectionViews.cells.otherElements.containing(.staticText, identifier: brick).children(matching: .other).element.tap()
-
-            app.pickerWheels.element.adjust(toPickerWheelValue: testValues[2])
-            app.buttons[kLocalizedDone].tap()
 
             app.collectionViews.cells.otherElements.containing(.staticText, identifier: brick).children(matching: .other).element.tap()
 
             app.pickerWheels.element.adjust(toPickerWheelValue: testValues[0])
```

Comment:
First of all, thank you for your PR!
Why do you use testValues[0] for the 'Done' and the 'Cancel' button?
Will this really show that the buttons works as intended? ;)

Reply:
Yes, you are right it should be `testValues[2]` in `line 106`.